lib/commands/actions.js

```diff
@@ -182,6 +182,41 @@ commands.sendSMS = async function (phoneNumber, message) {
   await this.adb.sendSMS(phoneNumber, message);
 };
 
+commands.gsmCall = async function (phoneNumber, action) {
```

Comment:
*adb* doesnt return any value when calling it, so I guess not. If adbExec fails it will throw an error, right ?